src/Entity/MediaBundle.php

```diff
@@ -174,4 +182,18 @@ protected function typePluginCollection() {
     return $this->typePluginCollection;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function isNewRevision() {
```

Comment:
This and function below are not defined by any interface that MediaBundle implements. They should be added to MediaBundleInterface.